.github/workflows/develop_freeze_comment.yml

```diff
@@ -1,5 +1,5 @@
-# This workflow will comment on Pull Requests that are submitted while develop
-# is frozen during the week of release. Skips any Pull Requests that have the
+# This workflow will comment on pull tequests that are submitted while develop
```

Comment:
Am I reading the diff backward or is this going from no typo to typo?

Reply:
whoops. Fixed.